### ISSUE TITLE
Outputs corrupt jars when in a spring boot executable project

### ISSUE THREAD

**Issue by adeptues** [Zlika/reproducible-build-maven-plugin#42]
Hi,
When running the strip-jar command after maven has build the jar file for the spring boot project the jar is corrupt when the following configuration is added to the spring boot plugin.
```xml
  <plugin>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-maven-plugin</artifactId>
                <configuration>
                    <executable>true</executable>
                </configuration>
            </plugin>
            <plugin>
                <groupId>io.github.zlika</groupId>
                <artifactId>reproducible-build-maven-plugin</artifactId>
                <version>0.10</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>strip-jar</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```

this  only occurs when the executable field is set

**Comment by Zlika** [comment]
Hi,
Thank you for reporting this issue. Can you give a sample oroject to reproduce this issue? Do you know what is the problem with the jar?

**Comment by adeptues** [comment]
Hi, project to reproduce is here https://github.com/adeptues/boot-reproducible-build the ouput is always somthing like 12k in size. The spring boot build puts an executable script at the front of the jar during its build process not sure if this affects it.

**Comment by Zlika** [comment]
Thanks. I'll have a look at it next weekend.

**Comment by Zlika** [comment]
Hi!
I added a new feature to detect spring-boot generated "executable" JAR/WAR/ZIP files, split the launch script from the actual ZIP file, strip the ZIP file and then re-add the spring boot launch script. 
Can you compile the HEAD version of the plugin and test it on your spring boot project to check if it is ok now?

**Comment by adeptues** [comment]
Hi, Sorry for the delay in testing this. I can confirm this now works and fixes the issue.

Any plans to release a new version?

**Comment by Zlika** [comment]
I've just released version 0.11.
